doc/source/multivisit_products.rst

```diff
@@ -0,0 +1,243 @@
+    MEDNEXP        Median number of HST exposure for the pixels which have been observed by HST
+    ===========    ================================================================================
+
+In addition, some keywords typically found in standard pipeline product headers or the headers of SVM mosaics have
+been removed from the headers of MVM products.  These deleted keywords can be found in the HDRTAB extension for each
+of the input exposures, but make no sense for MVM products.  One example would be the 'IPPPSSOO' keyword which
+gives the 'ipppssoo' value for a single exposure or association product the input exposures had, yet MVM mosaics
+consist of multiple inputs with many different values of 'ipppssoo'.  The full list of keywords which were removed can
+be found in the HISTORY keywords of the MVM product PRIMARY header where the rules file used for defining the MVM
+headers gets reported.
+
+
+Artifacts
+==========
+There are a number of issues that can arise when generating SkyCell mosaics.  Every effort is made during pipeline
+processing to minimize or avoid these issues where possible, but some mosaics are unavoidably affected by these issues.
+
+
+Mis-alignment
+--------------
+One of the primary benefits of SkyCell mosaic image is learning how observations taken at different times and using
+different filters relate to each other.  The mosaics all share the same pixel definitions which allow for direct
+comparisons of the data across all the layers of a SkyCell.  However, the placement of the exposures in the SkyCell
+depends on how the WCS was defined for each exposure.  Unfortunately, due to the objects in the field of view for an
+exposure or the size of the field of view of the exposure, it may not be possible to align the exposure to the same
+astrometric catalog as the rest of the exposures in the SkyCell.  This can lead to mis-alignment between the exposures
+on the order of a few pixels.  If this mis-aligned exposure overlaps another exposure aligned to GAIA in a SkyCell mosaic,
+then it can result in visible blurring or double-images in the final mosaic.
+
+This can be seen when examining the SkyCell mosaic WFC3/UVIS F555W layer for SkyCell **p0498x16y19**.  The WCS has been
+defined based on different astrometric catalogs for one of the visits of this source.  This results in
+'blurred' sources being seen in the overlap between exposures from different visits.
+
+.. figure:: images/mvm_p0498x16y19_f555w.png
+         :figwidth: 95%
+         :alt: SkyCell p0498x16y19 WFC3/UVIS F555W layer.
+
+         All the WFC3/UVIS F555W exposures that overlap SkyCell **p0498x16y19** illustrating
+         how misalignment between visits can result in 'blurred' sources in region of overlap.
+         [Image courtesy of J. Mack (STScI/ACS Instrument team)].
+
+
+Loss of Lock
+--------------
+Another issue that can show up in SkyCell mosaics results from HST slewing across the sky while the exposures was
```

Comment:
Change "exposures was" ==> "exposure was".

Reply:
Fixed in 175ee20.